packages/devtools-kit/src/core/component/state/format.ts

```diff
@@ -73,20 +73,22 @@ export function formatInspectorStateValue(value, quotes = false) {
   else if (typeof value === 'string') {
     const typeMatch = value.match(rawTypeRE)
     if (typeMatch)
-      value = escape(typeMatch[1])
+      value = escapeString(typeMatch[1])
 
     else if (quotes)
-      value = `<span>"</span>${escape(value)}<span>"</span>`
+      value = `<span>"</span><span class="truncated-string-field">${escapeString(value)}</span><span>"</span>`
```

Comment:
We should avoid using UnoCSS here to ensure compatibility with other projects using ·@vue/devtools-kit·.

Reply:
It makes sense, I updated it to use a custom prop instead of hard code.